src/zzz_od/application/hollow_zero/lost_void/operation/interact/lost_void_bangboo_store.py

```
@@ -97,10 +97,16 @@ def buy_artifact_gold(self) -> OperationRoundResult:
                 return self.round_wait(status='向右滑动')
             return self.round_retry(status='未识别可购买藏品', wait=1)
 
+        # 规则日志只在第1轮和放行状态变化轮打印，其他轮次不重复刷
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)